pkg/ingestor/ingestor.go

```diff
@@ -92,8 +92,23 @@ func BulkIngest(ctx context.Context, docs []*processor.Document, graphqlEndpoint
 			return fmt.Errorf("unable to ingest doc tree: %v", err)
 		}
 		for i := range preds {
+			predicates[0].CertifyScorecard = append(predicates[0].CertifyScorecard, preds[i].CertifyScorecard...)
```

Comment:
so we might have to put a limit here on how many we transfer to the graphQL server. After a certain limit, send to the graphQL server and start collecting again.

Reply:
I put a limit at 5000 `assembler.IngestPredicates` merged but I have no strong opinion on how much it's a good value so feel free about suggesting a more appropriate value in case.